pki/certs.go

```diff
@@ -130,16 +131,21 @@ func IssueTestCertificate(
 		return CertificatePair{}, err
 	}
 
-	earliest := clk.Now()
-	latest := earliest.AddDate(0, 0, 90)
+	// For non sharded logs, submit 90 day certificates with notBefore = now.
+	notBefore := clk.Now()
+	notAfter := clk.Now().AddDate(0, 0, 90)
 
-	if windowStart != nil {
-		earliest = *windowStart
-		earliest.AddDate(0, 0, 1)
-	}
+	// For sharded logs, ensure our notAfter is within the shard
+	// by setting creating a cert that expires 1 second before windowEnd.
+	// This may create certificates that are longer than 825 days, but they
+	// are not meant to be BR compliant.
```

Comment:
This comment seems slightly at odds with the top level function comment that says: "we try to avoid cablint errors to avoid requiring log monitors special-case our submissions."

How do you feel about dropping this comment and updating the function comment to mention that we try to avoid cablint errors with the one caveat of long certificate lifetimes?